.github/workflows/memory-leak-testing.yml

```
@@ -16,6 +16,15 @@ jobs:
         with:
           node-version: ${{ vars.NODE_VERSION }}
 
+      - name: Cache pnpm dependencies
+        id: cache-pnpm-store
+        uses: actions/cache@v3
```

actionlint (1.7.4)

the runner of "actions/cache@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)